lib/saml/post_url_service.rb

```diff
@@ -64,14 +64,22 @@ def tou_declined_logout_redirect_url
 
     def terms_of_use_redirect_url
       application = @tracker&.payload_attr(:application) || 'vaweb'
-      if TERMS_OF_USE_ENABLED_CLIENTS.include?(application)
+      if what_clients_are_enabled.include?(application)
```

Comment:
I think this check is more for which client is sending the request, like `vaweb` or one of the other clients.